diff --git a/Cargo.toml b/Cargo.toml
--- a/Cargo.toml
+++ b/Cargo.toml	(date 1707507666861)
@@ -1,13 +1,6 @@
 [package]
 name = "py-bip39-bindings"
-description = "Python bindings for tiny-bip39 RUST crate"
-authors=["Stichting Polkascan (Polkascan Foundation)"]
 version = "0.1.11"
-repository = "https://github.com/polkascan/py-bip39-bindings"
-homepage = "https://github.com/polkascan/py-bip39-bindings"
-license = "Apache-2.0"
-readme = "README.md"
-edition = "2018"

 [dependencies]
 hmac = "0.7.0"
@@ -22,17 +15,3 @@
 [dependencies.pyo3]
 version = "0.11.1"
 features = ["extension-module"]
-
-[package.metadata.maturin]
-maintainer = "Stichting Polkascan (Polkascan Foundation)"
-maintainer-email = "legal@polkascan.org"
-classifier = [
-    "License :: OSI Approved :: Apache Software License",
-    "Intended Audience :: Developers",
-    "Programming Language :: Python :: 3",
-    "Programming Language :: Python :: 3.7",
-    "Programming Language :: Python :: 3.8",
-    "Programming Language :: Python :: 3.9",
-    "Programming Language :: Python :: 3.10",
-    "Programming Language :: Python :: 3.11"
-]
diff --git a/pyproject.toml b/pyproject.toml
--- a/pyproject.toml	(date 1670928743000)
+++ b/pyproject.toml
@@ -2,3 +2,25 @@
 requires = ["maturin>=0.10,<0.11"]
 build-backend = "maturin"

+[project]
+name = "py-bip39-bindings"
+description = "Python bindings for tiny-bip39 RUST crate"
+authors=["Stichting Polkascan (Polkascan Foundation)"]
+version = "0.1.11"
+repository = "https://github.com/polkascan/py-bip39-bindings"
+homepage = "https://github.com/polkascan/py-bip39-bindings"
+license = "Apache-2.0"
+readme = "README.md"
+edition = "2018"
+maintainer = [{
+    name="Stichting Polkascan (Polkascan Foundation)", email="legal@polkascan.org"}]
+classifier = [
+    "License :: OSI Approved :: Apache Software License",
+    "Intended Audience :: Developers",
+    "Programming Language :: Python :: 3",
+    "Programming Language :: Python :: 3.7",
+    "Programming Language :: Python :: 3.8",
+    "Programming Language :: Python :: 3.9",
+    "Programming Language :: Python :: 3.10",
+    "Programming Language :: Python :: 3.11"
+]
